src/components/Buttons/FutureButton/FutureButton.tsx

```diff
@@ -23,12 +24,14 @@ const FutureButton: React.FC<FutureButtonProps> = ({
 	glow,
 	loading,
 	alt,
+	isWalletConnected,
```

Comment:
Noticed that this prop isn't being used anywhere. Make sure to prune out unused stuff.

Maybe a bit unrelated to this PR, but I noticed that this variable is acting like a boolean but it's typed as a string. Be careful to get your types correct - this wouldn't throw an error but it's confusing for other developers. You're passing the wallet ID from the parent, but you should be passing in a boolean with the value of `wallet ID !== undefined`. If a variable starts with "is" it should pretty much always be a boolean.

In terms of architecture, the buttons don't need any context for wallet state, etc. they just need to know if they should glow or not. Whether they should glow or not should be decided by the parent - this later is purely presentational. `glow` handles this functionality.